README.md

```diff
@@ -1 +1,222 @@
+
+##### Output
+
+| **Argument Name** | **Type** | **Description** |
+| --- | --- | --- |
+| Anomalous Objects Json | json  | The json object with holds the anomalous objects fetched from Cohesity Helios | 
+
+### <a name="restore-anomalous-objects"></a> Cohesity Helios - Restore Anomalous VM
+[top](#Cisco-SecureX-Integration)
+
+This Atomic Action performs a restore operation for the specified anomalous VM to latest clean snapshot. Currently only VMware VMs are supported
+
+##### Input
+
+| **Argument Name** | **Type** | **Description** | **Required** |
+| --- | --- |--- | --- |
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| VM Object | JSON | The VM object that needs to be restored   | Yes | 
+
+##### Output
+
+| **Argument Name** | **Type** | **Description** |
+| --- | --- | --- |
+| Restored VM Object  | JSON  | The restored VM object that was restored to the latest snapshot | 
+
+
+### <a name="ignore-anomaly"></a> Cohesity Helios - Ignore Anomaly
+[top](#Cisco-SecureX-Integration)
+
+This Atomic Action ignores the anomaly which removes the anomalous object from the list of anomalous object. 
+
+##### Input
+
+| **Argument Name** | **Type** | **Description** | **Required** |
+| --- | --- |--- | --- |
+| Cohesity Helios API Key | Secure String | API Key to access Helios | Yes | 
+| VM Object | JSON| The VM object that needs to be Ignored   | Yes | 
+
+##### Output
+
+N/A
+
+## <a name="workflows"></a> Workflows :hourglass_flowing_sand:
+[top](#Cisco-SecureX-Integration)
+
+Workflows are the larger component of orchestration and are similar to a script in traditional programming. A workflow can be simple and only have a few actions or be complex and string together many different actions for different products. Refer to the [Workflows](https://ciscosecurity.github.io/sxo-05-security-workflows/workflows/) documentation to find more. 
+
+> All the input to the workflows are passed as Global Variables.
+
+Lets go over the list of Workflows that this integration supports. 
+
+- [Helios Ransomware Alerts to SecureX and ServiceNow](#get-alerts-to-securex)
+- [Push Helios Ransomware Alerts to Private Intelligence](#push-alerts-to-pi)
+- [Ignore Anomaly on Cohesity Helios](#ignore-anomaly-workflow)
+- [Cohesity Restore Anomalous Object](#restore-anomaly)
+- [Push All Helios Ransomware Alerts to Private Intelligence](#push-all-anomaly)
+
+### <a name="get-alerts-to-securex"></a> Helios Ransomware Alerts to SecureX and ServiceNow 
+[top](#Cisco-SecureX-Integration)
+
+This workflow pushes Cohesity Helios ransomware alerts to Threat Response Private Intelligence data store and also creates incidents on ServiceNow at regular intervals based on the schedule you define. 
+
+This workflow can also be triggered based on a schedule. The schedule that is used here is `Cohesity Helios Ransomware Data Push Schedule` which is a schedule to trigger orchestration workflow to push Helios ransomware data to private intelligence.
+
```

Comment:
this workflow also needs creation of 'ServiceNow_Credentials' under Account Keys and 'Cohesity_ServiceNow_Target' under Targets. These are needed for creating incidents on service now